run_docker.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+rootdir=`dirname $0`
+pushd $rootdir
+docker build -t sdk-testing -f ./scripts/docker/Dockerfile $rootdir
```

Comment:
It would be better for us to put this in a docker registry than have people build it every time. This will make tests take longer to run and create a blocker when the dockerfile breaks (which it will eventually since people update packages, change urls and stuff)